code_samples/back_office/search/config/append_to_services.yaml

```
@@ -0,0 +1,5 @@
+services:
+
+    App\Search\SortingDefinition\Provider\SectionNameSortingDefinitionProvider:
+        tags:
+            -   name: ibexa.search.sorting_definition.provider
```

```suggestion
services:
    _defaults:
        autowire: true
        autoconfigure: true
        public: false
        
    App\Search\SortingDefinition\Provider\SectionNameSortingDefinitionProvider:
        tags:
            -   name: ibexa.search.sorting_definition.provider
```

Otherwise you have to manually provide ctor args (`@translator` in this case).

---

@adamwojs This is a yaml file sample. As it names suggest, it is to append to existing services.yaml which has those defaults already set.

---

Something like
`tail -n +2 $DOC/code_samples/back_office/search/config/append_to_services.yaml >> $DXP/config/services.yaml`

---

Side note, I'd like to establish a way to organize code samples so they could always be installed the same way for testing, something with CI as a further goal on mind.